cranelift/codegen/src/isa/aarch64/abi.rs

```diff
@@ -1271,38 +1271,44 @@ impl ABICall for AArch64ABICall {
 
     fn emit_call<C: LowerCtx<I = Self::I>>(&mut self, ctx: &mut C) {
         let (uses, defs) = (
-            mem::replace(&mut self.uses, Set::empty()),
-            mem::replace(&mut self.defs, Set::empty()),
+            mem::replace(&mut self.uses, Default::default()),
```

Comment:
Pre-existing, but I just realized that we don't document this side-effect of `emit_call` on the `ABICall` trait. Perhaps add to the doc comment (in the trait def) to note that `emit_call()` should only be called once, because it is allowed to re-use parts of the ABICall object in emitting instructions?

Reply:
Good point, added!